src/app/file-manager/warning-modal/warning-modal.component.html

```diff
@@ -9,7 +9,7 @@ <h2 mat-dialog-title>
 <mat-dialog-content>
   <div class="flex-container">
     <p>All current
-      <span *ngIf="data == 'exit' || data == 'logout'">transfers</span>
+      <span *ngIf="data == 'quit' || data == 'logout'">transfers</span>
```

Comment:
Is this an unrelated change?

Reply:
Phil, this change is a quick fix for develop, we can move this fix to a new branch if you think is better.

Reply:
It's ok just wanted to make sure it wasn't an accidental inclusion